### PR TITLE
IBL tutorials

### PR DESCRIPTION
**Pull request by oliche**
Added some basic tutorials for use at Neurodatarehack 2023

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .ipynb_checkpoints
+000409/paper-brain-wide-map/
```
Comment (bendichter) [inline .gitignore:2]: Was this supposed to be added here?

**Review by bendichter — CHANGES_REQUESTED** [review]
@oliche this is great! Would you mind adding a small README outlining the purpose of the different notebooks available?

**Comment by satra** [comment]
@oliche - could you also please structure the additions in the way the README describes: `<dandiset>/<orgorlab>/...`

**Comment by satra** [comment]
@oliche - @yarikoptic enabled codespell and there a few things that are being picked up that should be fixed. but there are others that are being misinterpreted. @yarikoptic - do you have any suggestions on best practice for people submitting notebooks. perhaps add that to the README of the repo.

**Comment by oliche** [comment]
Yeah but most of the errors are related to other notebooks.
As for my notebooks, it's not really working as the brain region acronyms and the logs are not amenable to spell checks.


```
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1863: ot ==> to, of, or, not
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1863: nOO ==> no
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1863: Ot ==> To, Of, Or, Not
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1863: yuO ==> you
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: te ==> the, be, we, to
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: bu ==> by, be, but, bug, bun, bud, buy, bum
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: bu ==> by, be, but, bug, bun, bud, buy, bum
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: OLL ==> ALL, OLE, OLD, OLLY, OIL
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: fo ==> of, for, to, do, go
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: fOf ==> for
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: te ==> the, be, we, to
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: te ==> the, be, we, to
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: te ==> the, be, we, to
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: uE ==> use, due
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: te ==> the, be, we, to
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: SoM ==> some
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: fo ==> of, for, to, do, go
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: fwe ==> few
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: whe ==> when, we
Error: ./000409/IBL/03_analysis_Imbizo_2023.ipynb:1915: te ==> the, be, we, to
``

**Review by bendichter — APPROVED** [review]

**Pull request merged**